### PR TITLE
Add alt text to images on the homepage

### PR DESCRIPTION
**Pull request by julianna-langston**
For this issue: https://github.com/chartjs/Chart.js/issues/11231

Adds alternative text to the images on the chart.js homepage.

### CODE REVIEW

**Review by etimberg — APPROVED** [review]

File: src/templates/footer.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <div class="footer-logo">
-	<img src="{{base_path}}/img/icon.svg"/>
+	<img src="{{base_path}}/img/icon.svg" alt="" />
```
Comment (LeeLenaleee) [inline src/templates/footer.html:2]: Logo grey scale would be a better fit as an empty alt text

File: src/templates/homepage.html [hunk 2]
```diff
@@ -159,36 +159,36 @@ <h3 class="subtitle">
...
+						<canvas id="animate-chart" height="200" width="300" aria-label="Circles animate as they change position, size, and color"></canvas>
 					</div>
 				</div>
 			</div>
 			<hr/>
 			<div class="feature-small-container fade-in animation-delay__7">
 				<div class="feature-small">
 					<div class="feature-icon">
-						<img src="./img/open-source.svg"/>
+						<img src="./img/open-source.svg" alt="" />
```
Comment (LeeLenaleee) [inline src/templates/homepage.html:170]: Open source indicator icon

File: src/templates/homepage.html [hunk 3]
```diff
@@ -159,36 +159,36 @@ <h3 class="subtitle">
...
 					</div>
 					<h4 class="subtitle subtitle__small">Open source</h4>
 					<p class="description description__small">Chart.js is a community maintained project, contributions welcome!</p>
 				</div>
 				<div class="feature-small">
 					<div class="feature-icon">
-						<img src="./img/chart-types.svg"/>
+						<img src="./img/chart-types.svg" alt="" />
```
Comment (LeeLenaleee) [inline src/templates/homepage.html:177]: Bar chart icon

File: src/templates/homepage.html [hunk 4]
```diff
@@ -159,36 +159,36 @@ <h3 class="subtitle">
...
 					</div>
 					<h4 class="subtitle subtitle__small">8 Chart types</h4>
 					<p class="description description__small">Visualize your data in 8 different ways; each of them animated and customisable.</p>
 				</div>
 				<div class="feature-small">
 					<div class="feature-icon">
-						<img src="./img/canvas-icon.svg"/>
+						<img src="./img/canvas-icon.svg" alt="" />
```
Comment (LeeLenaleee) [inline src/templates/homepage.html:184]: Html logo

File: src/templates/homepage.html [hunk 5]
```diff
@@ -159,36 +159,36 @@ <h3 class="subtitle">
...
 					</div>
 					<h4 class="subtitle subtitle__small">HTML5 Canvas</h4>
 					<p class="description description__small">Great rendering performance across all modern browsers (IE11+).</p>
 				</div>
 				<div class="feature-small">
 					<div class="feature-icon">
-						<img src="./img/responsive.svg"/>
+						<img src="./img/responsive.svg" alt="" />
```
Comment (LeeLenaleee) [inline src/templates/homepage.html:191]: Responsive icon

**Review by LeeLenaleee — CHANGES_REQUESTED** [review]
Would be better to have a description in all alt texts instead of having empty ones

**Comment by julianna-ciq** [comment]
I thought that those icons were all decorative. [Decorative images shouldn't have any alt text](https://www.w3.org/WAI/tutorials/images/decorative/), because it doesn't provide additional information to a screen reader user. For example, if we used the alt text "Responsive" for the icon above the text "Responsive", a screen reader will read that as:
> Responsive, graphic
> Responsive, heading level 4

In this case, the icon's alt text is redundant. 

If you consider the images to be *informative*, rather than decorative, I can certainly update the alt text.

**Review by LeeLenaleee — APPROVED** [review]

**Pull request merged**